build.gradle

```diff
@@ -214,6 +241,12 @@ dependencies {
     // testCompile group: 'com.alibaba', name: 'fastjson', version:'1.2.56'
     // testCompile group: 'org.mockito', name: 'mockito-core', version:'2.23.4'
     testCompile group: "org.elasticsearch.client", name: 'transport', version: "${es_version}"
+
+    // JDBC drivers for comparison test. Somehow Apache Derby throws security permission exception.
+    testCompile group: 'com.amazon.opendistroforelasticsearch.client', name: 'opendistro-sql-jdbc', version: '1.3.0.0'
```

Comment:
Just curious, are we going to base the test on v1.3? Because I thought it would be v1.4.

Reply:
Yes, JDBC driver 1.4 is not published to Maven central repository yet.